bundle/manifests/designer.kaoto.io_kaotoes.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.1
+  creationTimestamp: null
+  name: kaotoes.designer.kaoto.io
+spec:
+  group: designer.kaoto.io
+  names:
+    kind: Kaoto
+    listKind: KaotoList
+    plural: kaotoes
+    singular: kaoto
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - description: The phase
+      jsonPath: .status.phase
+      name: Phase
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Kaoto is the Schema for the kaotoes API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: KaotoSpec defines the desired state of Kaoto
+            properties:
+              image:
+                default: docker.io/kaotoio/standalone:main
+                type: string
+            type: object
+          status:
+            description: KaotoStatus defines the observed state of Kaoto
+            properties:
+              conditions:
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
Need to update the base version of this description.

Reply:
what do you mean here ?